server/authn/presharedkey/presharedkey.go

```diff
@@ -5,8 +5,15 @@ import (
 
 	"github.com/go-errors/errors"
 	grpcAuth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
-
 	"github.com/openfga/openfga/server/authn"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	ErrMissingBearerToken = status.Error(codes.Code(openfgapb.AuthErrorCode_bearer_token_missing), "missing bearer token")
```

Comment:
Can we put these overlapping definitions (here and above in the OIDC authenticator) in the top-level shared package. `authn/authn.go` file 

That way we don't have duplication.

Reply:
Good call. Thanks! Updated.